org-code-javabuilder/playground/src/main/java/org/code/playground/Board.java

```diff
@@ -0,0 +1,124 @@
+package org.code.playground;
+
+import java.io.FileNotFoundException;
+import org.code.media.Color;
+import org.code.media.Font;
+import org.code.media.FontStyle;
+
+public class Board {
+    /**
+     * Returns the width of the playground screen. This will always be 400.
+     * 
+     * @return the width of the playground in pixels.
+     */
+    public int getWidth();
+
+    /**
+     * Returns the height of the playground screen. This will always be 400.
+     * 
+     * @return the height of the playground in pixels.
+     */
+    public int getHeight();
+
+    /**
+     * Sets the background of the playground to the provided image. The image will
+     * be scaled to fit the full playground screen, which may distory the image if
+     * it is not square.
+     * 
+     * @param filename the name of the file from the asset manager to put in the
+     *                 background
+     * @throws FileNotFoundException if the file cannot be found in the asset
+     *                               manager
+     */
+    public void setBackgroundImage(String filename) throws FileNotFoundException;
+
+    /**
+     * Adds a clickable item from the playground.
+     * 
+     * @param item the item to add. If the item is already in the playground,
+     *              this method does nothing.
+     * @param x the distance, in pixels, from the left side of the playground
+     * @param y the distance, in pixels, from the top of the playground
+     */
+    public void addItem(ClickableItem item, int x, int y);
+
+    /**
+     * Removes the clickable item from the playground.
+     * 
+     * @param item the item to remove. If the item is not in the playground, this
+     *              method does nothing.
+     */
+    public void removeClickableItem(ClickableItem item);
```

Comment:
Since `ClickableItem` extends `Item`, does it make sense to just have one `addItem` and one `removeItem` method that takes the generic `Item`?

Reply:
Can you easily delineate between the two in the implementation? I thought downcasting was a pain in Java...

Reply:
That's fair, it would make it easier on us to have both. If a student passes in a `ClickableItem` to `addItem` should we not support clicking it?

Reply:
Yes, anything passed into the `addItem` method, event if it's a `ClickableItem`, should not have any click events registered.